### PR TITLE
Support for :decimal 2.0 dependency

### PR DESCRIPTION
**Pull request by patrykwozinski**

### CODE REVIEW

**Comment by xavier** [comment]
Thanks! 👍 

Before we can really claim that this library supports Decimal 2.0, there's a small change to be made to `XlsxReader.Conversion.to_decimal/1` to support the value returned by `Decimal.parse/1` in Decimal 2.0.

Instead of returning `{:ok, value}`, it now returns `{value, binary}` like  `Float.parse/1` or `Integer.parse/2`.

We should make sure our conversion function behaves consistently with both Decimal 1.x and Decimal 2.x and returns our expected success tuple `{:ok, value}`.

**Comment by patrykwozinski** [comment]
@xavier thanks a lot for a fast answer. I've added this small change to keep consistency with the return of the `to_decimal` function.

4e9692d387f1fcc4c0dfd76ae16059a24285d8ff <- please verify if it's okay in your standard (I'm Elixir newbie).

File: lib/xlsx_reader/conversion.ex [hunk 1]
```diff
@@ -151,7 +151,11 @@ defmodule XlsxReader.Conversion do
   """
   @spec to_decimal(String.t()) :: {:ok, Decimal.t()} | :error
   def to_decimal(string) do
-    Decimal.parse(string)
+    case Decimal.parse(string) do
+      {:ok, decimal} -> {:ok, decimal}
+      {decimal, _} -> {:ok, decimal}
```
Comment (xavier) [inline lib/xlsx_reader/conversion.ex:156]: To be consistent with our other conversion functions, we should match strictly on `{decimal, ""}` to reject any decimal with trailing characters.

Comment (patrykwozinski) [reply lib/xlsx_reader/conversion.ex]: Sure, changed! :) I've also reformatted these lines to your library coding standard. :)

**Review by xavier — APPROVED** [review]

**Comment by xavier** [comment]
👍

**Pull request merged**